api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -71,8 +88,16 @@
 
   @Override
   public ResponseEntity<List<BillingProjectMembership>> getBillingProjects() {
-    // TODO: retrieve billing projects
-    return ResponseEntity.ok(new ArrayList<>());
+    try {
```

Comment:
From your background on Slack -- maybe a brief comment here saying that we expect this to always be the shared free-tier billing project, until <some change>?

Reply:
So that's not *quite* accurate. Somebody could go into Firecloud and create additional projects with the FC user we created for their researcher account, and those would show up in this list, too.

It will be up to the client to look specifically for the free tier billing project for now.

Reply:
(We might in future decide to filter projects returned here down to those that were created via AofU... but right now no logic like that is in place.)